desktop/cmp/rest/RestGridModel.js

```diff
@@ -80,6 +80,7 @@ export class RestGridModel {
     get selectedRecord() {return this.gridModel.selectedRecord}
 
     /**
+     * @param {boolean} [editable] - make CRUD controls visible. Defaults to true.
```

Comment:
Make this description more general -- e.g. allow 'user to edit/update/delete data.'
 
The default toolbars are probably only part of the story, and any app can always throw in whatever toolsbars they want -- for consistency, we should probably throw in RecordForm.saveRecord and RecordGridModel.deleteRecord() if somebody tries to edit a non-editable form.